### ISSUE TITLE
Jiaxin_code_review_technology selection

### ISSUE THREAD

**Issue by manymuch** [plantclassification/seedlings_classification#5]
For cnn model, vgg is a good classification model and easy to train.  After training, I suggest there cloud be a threshold to control the output label when the user input is no belong to the 11 classes defined in the model.

**Comment by manymuch** [comment]
BTW, since the dataset is not very big, you can use some random prepocessing for images like sheer, flip, whitenning adjusement. Making some noises on original datasets to create more images, it can also help to reduce overfitting effect during training. And use less dropout layers.
